lib/node-integration-iframe.mustache.js

```diff
@@ -1,4 +1,8 @@
-(function () {
+(function (runtimeFlag) {
+  if (window[runtimeFlag] === true) {
```

Comment:
Unfortunately `runInParent` supercedes `useIframe` so even if we were to specify it, `runInParent` will still win out (i.e. all code is executed in main Karma context instead of a separate iframe/window)